packages/e2e-test-utils/src/disable-experimental-features.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Internal dependencies
+ */
+import { setExperimentalFeaturesState } from './enable-experimental-features';
+
+/**
+ * Disables experimental features from the plugin settings section.
+ *
+ * @param {Array} features Array of {string} selectors of settings to enable. Assumes they can be enabled with one click.
+ */
+export async function disableExperimentalFeatures( features ) {
```

Comment:
It feels weird to have `setExperimentalFeaturesState` and  `enable` in one file but `disable` in another one. We should probably move them all together and rename the file something like `experimental-features`. I also think this should not be part of ` e2e-test-utils` and should be moved to the `experimental` folder of th `e2e-tests`

Reply:
Right, I didn't think about it at all. 100% agree!

Reply:
Hi @youknowriad, @gziolo, we are exposing enableExperimentalFeatures in the e2e-test-utils package. This util is just used for tests and not used in production code. In this case, we don't need to keep it in e2e-test-utils, and we can remove it from the package, is that right?

Reply:
yes, it's a dev package,  breaking changes are fine.